src/net.cpp

```diff
@@ -148,36 +148,33 @@ uint16_t GetListenPort()
 // find 'best' local address for a particular peer
 [[nodiscard]] static bool GetLocal(CService& addr, const CNode& peer)
 {
-    if (!fListen)
-        return false;
+    if (!fListen) return false;
 
-    int nBestScore = -1;
-    int nBestReachability = -1;
+    int best_reachability = -1;
+    int best_score = -1;
```

Comment:
nit: not sure the snake case is worth the added LoC (+further down). Would prefer to keep as is.